### PR TITLE
anchore-cli task

### PR DESCRIPTION
**Pull request by arunvel1988**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Follows the [authoring recommendations](https://github.com/tektoncd/catalog/blob/main/recommendations.md)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality of task changed or new task added)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [ ] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [ ] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [ ] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [ ] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [ ] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [ ] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

---

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]:  https://github.com/tektoncd/catalog/issues/413

### CODE REVIEW

**Comment by vinamra28** [comment]
> [CLA Missing ID](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/8049964/183500906/976/#/?version=2) [CLA Not Signed](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/8049964/183500906/976/#/?version=2)
> 
> * ❌ The commit ([fcc7073](https://github.com/tektoncd/catalog/commit/fcc70736dc886acef0f00eaef309e8f9eff0a1a7)). This user is missing the User's ID, preventing the EasyCLA check. [Consult GitHub Help](https://help.github.com/en/github/committing-changes-to-your-project/why-are-my-commits-linked-to-the-wrong-user) to resolve.For further assistance with EasyCLA, [please submit a support request ticket](https://jira.linuxfoundation.org/servicedesk/customer/portal/4).

@arunvel1988 can you please sign the CLA?

File: task/bentoml/anchore-cli/0.1/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+The Anchore CLI provides a command line interface on top of the Anchore Engine REST API.
+Anchore CLI will try to connect to the Anchore Engine at http://localhost/v1 with no authentication. The username, password and URL for the server can be passed to the Anchore CLI as command line arguments
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/anchore-cli/0.1/anchore-cli.yaml
+```
+
```
Comment (vinamra28) [inline task/bentoml/anchore-cli/0.1/README.md:11]: any reason for placing the `anchore-cli` task in `bentoml` task ?

File: task/anchore-cli/0.2/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+# Anchore CLI
+
+The Anchore CLI provides a command line interface on top of the Anchore Engine REST API.
+Anchore CLI will try to connect to the Anchore Engine at http://localhost/v1 with no authentication. The username, password and URL for the server can be passed to the Anchore CLI as command line arguments
+
+## Install the Task
```
Comment (vinamra28) [inline task/anchore-cli/0.2/README.md:6]: any reason for adding 2 versions of the same task?

File: task/bentoml/anchore-cli/0.1/README.md (hunk 1 above)
Comment (arunvel1988) [reply task/bentoml/anchore-cli/0.1/README.md:11]: no.
May be file was added accidentally

File: task/anchore-cli/0.2/README.md (hunk 2 above)
Comment (arunvel1988) [reply task/anchore-cli/0.2/README.md:6]: first version I had passed ANCHORE USERNAME and Password as env variables
second version refers to using secret

File: task/bentoml/anchore-cli/0.1/README.md (hunk 1 above)
Comment (vinamra28) [reply task/bentoml/anchore-cli/0.1/README.md:11]: @arunvel1988 in that case, can you please remove it?

File: task/anchore-cli/0.2/README.md (hunk 2 above)
Comment (vinamra28) [reply task/anchore-cli/0.2/README.md:6]: Can we merge both? Also, IMHO referring via secrets would be a better option but I'll leave it upto you.

Comment (arunvel1988) [reply task/anchore-cli/0.2/README.md]: @vinamra28  have merged

File: task/bentoml/anchore-cli/0.1/README.md (hunk 1 above)
Comment (arunvel1988) [reply task/bentoml/anchore-cli/0.1/README.md]: @vinamra28  removed

File: task/anchore-cli/0.2/README.md (hunk 2 above)
Comment (arunvel1988) [reply task/anchore-cli/0.2/README.md]: @vinamra28 any update

**Comment by arunvel1988** [comment]
@vinamra28 I have completed EasyCLA.

**Comment by arunvel1988** [comment]
why is this still open? I have completed EasyCLA

Should I commit once again? @vinamra28 
Can you help here

**Comment by vinamra28** [comment]
@arunvel1988, I guess you'll have to repush. Can you please do one more thing, can you please squash the commits also into one commit?
/ok-to-test

File: task/anchore-cli/0.1/anchore-cli.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    The Anchore CLI provides a command line interface on top of the Anchore Engine REST API.
+    Using the Anchore CLI users can manage and inspect images, policies, subscriptions and registries
+  workspaces:
+    - name: manifest-dir
+  params:
+    - name: ARGS
+      description: The Arguments to be passed to anchore command.
+      type: array
+    - name: ANCHORE_CLI_IMAGE
+      default: anchore/engine-cli
```
Comment (vinamra28) [inline task/anchore-cli/0.1/anchore-cli.yaml:25]: can you please tag the image as well with it's appropriate tag and digest? (Reason: The integrity of Task should remaining the same everytime somebody uses the Task)

Comment (arunvel1988) [reply task/anchore-cli/0.1/anchore-cli.yaml:25]: sure will do it

**Pull request closed without merging**